packages/radio-group/src/vaadin-radio-group.d.ts

```diff
@@ -17,144 +20,93 @@ export type RadioGroupInvalidChangedEvent = CustomEvent<{ value: boolean }>;
  *
  * The following state attributes are available for styling:
  *
- * Attribute  | Description | Part name
- * -----------|-------------|------------
- * `disabled`   | Set when the radio group and its children are disabled. | :host
- * `readonly` | Set to a readonly radio group | :host
- * `invalid` | Set when the element is invalid | :host
- * `has-label` | Set when the element has a label | :host
- * `has-value` | Set when the element has a value | :host
- * `has-helper` | Set when the element has helper text or slot | :host
- * `has-error-message` | Set when the element has an error message, regardless if the field is valid or not | :host
- * `focused` | Set when the element contains focus | :host
+ * Attribute           | Description                               | Part name
+ * --------------------|-------------------------------------------|------------
+ * `disabled`          | Set when the element is disabled          | :host
+ * `readonly`          | Set when the element is readonly          | :host
+ * `invalid`           | Set when the element is invalid           | :host
+ * `focused`           | Set when the element is focused           | :host
+ * `has-label`         | Set when the element has a label          | :host
+ * `has-value`         | Set when the element has a value          | :host
+ * `has-helper`        | Set when the element has helper text      | :host
+ * `has-error-message` | Set when the element has an error message | :host
  *
  * See [Styling Components](https://vaadin.com/docs/latest/ds/customization/styling-components) documentation.
  *
  * @fires {CustomEvent} invalid-changed - Fired when the `invalid` property changes.
  * @fires {CustomEvent} value-changed - Fired when the `value` property changes.
  */
-declare class RadioGroupElement extends ThemableMixin(DirMixin(HTMLElement)) {
+declare class RadioGroup extends FieldMixin(
+  FocusMixin(DisabledMixin(KeyboardMixin(DirMixin(ThemableMixin(HTMLElement)))))
+) {
   /**
-   * The current disabled state of the radio group. True if group and all internal radio buttons are disabled.
-   */
-  disabled: boolean | null | undefined;
-
-  /**
-   * This attribute indicates that the user cannot modify the value of the control.
-   */
-  readonly: boolean | null | undefined;
-
-  /**
-   * This property is set to true when the value is invalid.
-   */
-  invalid: boolean;
-
-  /**
-   * Specifies that the user must fill in a value.
-   */
-  required: boolean | null | undefined;
-
-  /**
-   * Error to show when the input value is invalid.
-   * @attr {string} error-message
-   */
-  errorMessage: string;
-
-  /**
-   * String used for the label element.
-   */
-  label: string;
-
-  /**
-   * String used for the helper text.
-   * @attr {string} helper-text
-   */
-  helperText: string | null;
-
-  /**
-   * Value of the radio group.
+   * The value of the radio group.
+   *
+   * @attr {value} readonly
```

Comment:
This annotation is incorrect, please remove it. We only specify `@attr` for camel-case properties that have dash-case attributes e.g. `errorMessage` / `error-message`, `helperText` / `helper-text`.

Reply:
Got it, thanks!